storage/maria/unittest/CMakeLists.txt

```diff
@@ -133,4 +133,3 @@ MY_ADD_TEST(ma_pagecache_rwconsist_1k)
 ADD_EXECUTABLE(ma_pagecache_rwconsist2_1k-t ma_pagecache_rwconsist2.c)
 SET_TARGET_PROPERTIES(ma_pagecache_rwconsist2_1k-t PROPERTIES COMPILE_FLAGS "-DTEST_PAGE_SIZE=1024")
 MY_ADD_TEST(ma_pagecache_rwconsist2_1k)
-
```

Comment:
Here, the change is just whitespace. You can remove the file from commit list, I guess.